lib/ui/src/components/preview/FramesRenderer.tsx

```diff
@@ -12,6 +13,23 @@ const getActive = (refId: FramesRendererProps['refId']) => {
   return 'storybook-preview-iframe';
 };
 
+const SkipToSidebarLink = styled(Button)(({ theme }) => ({
+  display: 'none',
+  '@media (min-width: 600px)': {
+    display: 'block',
+    position: 'absolute',
+    top: '10px',
+    right: '15px',
+    padding: '10px 15px',
+    fontSize: theme.typography.size.s1,
```

Comment:
Be aware in styled-components (e.g. in Chromatic), numbers will use the standard unit for the property, which is `em` for font-size. In Emotion it will always use `px` except for unitless properties (e.g. z-index), so in this case it works out.